packages/neos-ui/src/manifest.dataloaders.js

```diff
@@ -221,7 +221,7 @@ manifest('main.dataloaders', {}, globalRegistry => {
             }).then(() => {
                 // Trigger query
                 const assetProxySearchApi = backend.get().endpoints.assetProxySearch;
-                const resultPromise = assetProxySearchApi(searchTerm, '', options);
+                const resultPromise = assetProxySearchApi(searchTerm, options);
```

Comment:
@Sebobo asked (in #2757) why this param can be removed.
=> This was referring to the `assetSourceIdentifier` param, that is now replaced by `options.constraints.assetSources` (See https://github.com/neos/neos-ui/pull/2759/files#diff-d34d33e9dc47b1d77ad5f29e5a091011R260)

Thinking about it, it probably makes sense to keep it though for b/c..

Reply:
OTOH, is that really part of the public API?

Reply:
I re-added it, you never know...